fdbserver/Resolver.actor.cpp

```diff
@@ -388,22 +388,27 @@ ACTOR Future<Void> resolveBatch(Reference<Resolver> self,
 		auto& stateTransactions = stateTransactionsPair.second;
 		int64_t stateMutations = 0;
 		int64_t stateBytes = 0;
-		std::unique_ptr<LogPushData> toCommit(nullptr); // For accumulating private mutations
-		std::unique_ptr<ResolverData> resolverData(nullptr);
+		const bool useResolverPrivateMutations = SERVER_KNOBS->PROXY_USE_RESOLVER_PRIVATE_MUTATIONS;
+		const bool applyResolverPrivateMutations = useResolverPrivateMutations && !req.txnStateTransactions.empty();
+
+		std::unique_ptr<LogPushData> toCommit; // For accumulating private mutations
```

Comment:
These pointers are not initialized with "nullptr" now - is that an intentional change?

Reply:
removed redundant code.. in C++11 and later, std::unique_ptr is automatically initialized to nullptr by default ([link](https://en.cppreference.com/w/cpp/memory/unique_ptr/unique_ptr))